lib/src/network/data/notification/notification_repository_mock.dart

```diff
@@ -1,95 +1,84 @@
-import 'package:google_maps_flutter/google_maps_flutter.dart';
-import 'package:myapp/src/network/model/common/pagination/meta/pagination_meta.dart';
-import 'package:myapp/src/network/model/common/pagination/pagination.dart';
-import 'package:myapp/src/network/model/common/pagination/pagination_response.dart';
-import 'package:myapp/src/network/model/common/result.dart';
-import 'package:myapp/src/network/model/event/event.dart';
-import 'package:myapp/src/network/model/notification/change_event.dart';
-import 'package:myapp/src/network/model/notification/follow_event.dart';
-import 'package:myapp/src/network/model/notification/follow_user.dart';
-import 'package:myapp/src/network/model/notification/notification_model.dart';
-import 'package:myapp/src/network/model/notification/upcoming_event.dart';
-import 'package:myapp/src/network/model/user/user.dart';
+// import 'package:google_maps_flutter/google_maps_flutter.dart';
```

Comment:
After update repository implement, you can remove mock repo, don't need comment code here